workflow.rst

```diff
@@ -458,7 +458,7 @@ This means that each event has access to the following information:
     Returns a metadata.
 
 For Guard Events, there is an extended :class:`Symfony\\Component\\Workflow\\Event\\GuardEvent` class.
-This class has two more methods:
+This class has the additonal methods:
```

Comment:
`the additional methods` doesn't sound right to me ... `these additional methods` ?